pkg/controller/machine-set-boot-image/machine_set_boot_image_controller.go

```diff
@@ -276,19 +286,27 @@ func (ctrl *Controller) addConfigMap(obj interface{}) {
 
 // TODO: This callback happens every ~15 minutes or so even if the configmap contents is not updated
 // Perhaps worth implementing a diff check before starting the sync?
-func (ctrl *Controller) updateConfigMap(old, _ interface{}) {
+func (ctrl *Controller) updateConfigMap(old, new interface{}) {
```

Comment:
suggestion (non-blocking): `new` is a keyword in Golang. For clarity, you may want to use `oldCM` / `newCM` instead.

Reply:
Will do!